docs/next/guides/building-and-testing/packages.md

```diff
@@ -29,12 +29,12 @@ You can also use minified files:
 If you are a "Bob the Builder" kind of hacker, you will need to load Handsontable JS, CSS and their dependencies:
 ```html
 <!-- Required dependencies (as external scripts) -->
-<link href="https://cdn.jsdelivr.net/npm/pikaday@1.5.1/css/pikaday.min.css" rel="stylesheet">
+<link href="https://cdn.jsdelivr.net/npm/pikaday@1.8.0/css/pikaday.min.css" rel="stylesheet">
 <script src="https://cdn.jsdelivr.net/npm/moment@2.29.3/moment.min.js"></script>
-<script src="https://cdn.jsdelivr.net/npm/pikaday@1.5.1/pikaday.min.js"></script>
-<script src="https://cdn.jsdelivr.net/npm/numbro@2/dist/numbro.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/pikaday@1.8.0/pikaday.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/numbro@2.1.2/dist/numbro.min.js"></script>
 <script src="https://cdn.jsdelivr.net/npm/hot-formula-parser@3.0.0/dist/formula-parser.min.js"></script>
```

Comment:
Please remove `hot-formula-parser` from lines 36 and 44. It is not needed since https://github.com/handsontable/handsontable/blob/1db57ea486b4ff7fc3cf6e418d03da88900f2cc6/docs/next/guides/upgrade-and-migration/release-notes.md#L192

Reply:
Done [3f93f78](https://github.com/handsontable/handsontable/pull/9271/commits/3f93f78b80b92ef054bc8215f346e8cbf7d39e26)